docker/docker/src/main/java/org/arquillian/cube/docker/impl/docker/DockerClientExecutor.java

```diff
@@ -554,10 +556,15 @@ public void pullImage(String imageName) {
 
     public String execStart(String containerId, String... commands) {
         ExecCreateCmdResponse execCreateCmdResponse = this.dockerClient.execCreateCmd(containerId)
-                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty().withCmd(commands)
-                .exec();
-        InputStream consoleOutputStream = dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                .withAttachStdout(true).withAttachStdin(false).withAttachStderr(false).withTty(false).withCmd(commands)
                 .exec();
+        try {
+            dockerClient.execStartCmd(execCreateCmdResponse.getId()).withDetach(false)
+                    .exec(new ExecStartResultCallback(System.out, System.err)).awaitCompletion();
+        } catch (InterruptedException e) {
+            return "";
+        }
+        InputStream consoleOutputStream = dockerClient.copyArchiveFromContainerCmd(containerId, "/execStartTest.log").exec();
```

Comment:
Or actually, since we read in the whole input stream as a String and return it, we might as well pass in a ByteArrayOutputStream to the ExecStartResultCallback and return byteArrayOutput.toString()